src/server/components/formRunner/__tests__/helpers.test.ts

```diff
@@ -523,30 +514,15 @@ describe("Form Runner Service:", () => {
     };
 
     test("generated form runner webhook data is correct", async () => {
-      const isUnderTest = true;
```

Comment:
isUnderTest is no longer necessary. it should generally be avoided too. Code should be written in a testable way where you don't pass any flags in.

Reply:
What has changed now that makes the isUnderTest no longer necessary?

Reply:
the tsconfig (and jest config) now includes the forms directory and aliases it to @forms/* https://github.com/UKForeignOffice/lists/pull/614/files/dd48beb48b5bbb03abc2fe9d84eb262266756ab3#r1183401952

`isUnderTest` was detecting if `dist` was in __dirname - and then doing a string replace on dir/path when trying to do a `fs.promises.readFile`.

Reply:
Ah I've just noticed this. Looks like the project convention is to not have aliases with an `@`. Not sure if it makes sense to go with one way or the other just to keep things consistent.